g2p/apply.py

```diff
@@ -16,7 +16,10 @@ class ApplyG2PModelJob(Job):
     Apply a trained G2P on a word list file
     """
 
-    __sis_hash_exclude__ = {"filter_empty_words": False}
+    __sis_hash_exclude__ = {
+        "filter_empty_words": False,
+        "concurrent": 1,
```

Comment:
Ideally the concurrency should not change the result and therefore be excluded from hash computation.

Reply:
Solved now by implementing a custom hash function.